optout/pom.xml

```diff
@@ -22,15 +22,19 @@
             <version>0.0.1-SNAPSHOT</version>
         </dependency>
         <dependency>
-            <groupId>org.quartz-scheduler</groupId>
-            <artifactId>quartz</artifactId>
-            <version>${quartz.version}</version>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-quartz</artifactId>
```

Comment:
switching over to use spring-boot starter for quarter to get the default configuration as described [here](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-quartz)

Reply:
👍